pgbedrock/core_configure.py

```diff
@@ -23,6 +24,11 @@
 FILE_OPEN_ERROR_MSG = "Unable to open file '{}':\n{}"
 MISSING_ENVVAR_MSG = "Required environment variable not found:\n{}"
 VALIDATION_ERR_MSG = 'Spec error: role "{}", field "{}": {}'
+MULTIPLE_SCHEMA_OWNER_ERR_MSG = 'Spec error: schema "{}" owned by more than one role: {}'
+DUPLICATE_ROLE_DEFINITIONS_ERR_MSG = 'Spec error: role(s) defined more than once: {}'
+OBJECT_REF_READ_WRITE_ERR = ('Spec error: objects have been unnecessarily given both read and write privileges.'
+                             'pgbedrock automatically grants read access when write access is requested. {}\n'
```

Comment:
This should end with `requested.\n\t{}` so that the first entry shows up on a new line and tabbed in like all subsequent entries will.